lima.yaml

```diff
@@ -6,7 +6,8 @@
 # use vz instead of qemu for rosetta support
 vmType: "vz"
 
-cpus: 6
+cpus: 8
+memory: 16GiB
```

Comment:
Roughly equivalent to a CI image of `xlarge` https://circleci.com/docs/configuration-reference/